1-exercises/A-undefined/exercise.js

```diff
@@ -20,17 +20,17 @@ function sayHello() {
 }
 
 let hello = sayHello();
-console.log(hello);
+console.log(hello); // function is not called & hello variable is not declared with the string
```

Comment:
In this one, the function is being called on the previous line, but the problem is that the function is not returning anything.